src/Admin/PluginRowMeta.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Class PluginRowMeta.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Delayed;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * Updates the plugin row meta for the plugin.
+ *
+ * @since 2.1
+ * @internal
+ */
+final class PluginRowMeta implements Delayed, Service, Registerable {
+
+	/**
+	 * Get the action to use for registering the service.
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'admin_init';
+	}
+
+	/**
+	 * Runs on instantiation.
+	 */
+	public function register() {
+		add_filter( 'plugin_row_meta', [ $this, 'get_plugin_row_meta' ], 10, 2 );
+	}
+
+	/**
+	 * Updates the plugin row meta with links to review plugin and get support.
+	 *
+	 * @param string[] $meta        An array of the plugin's metadata, including the version, author, author URI,
+	 *                              and plugin URI.
+	 * @param string   $plugin_file Path to the plugin file relative to the plugins directory.
+	 * @return string[] Plugin row meta.
+	 */
+	public function get_plugin_row_meta( $meta, $plugin_file ) {
+		if ( plugin_basename( AMP__FILE__ ) !== $plugin_file ) {
+			return $meta;
+		}
+
+		$additional_meta = [
+			'<a href="https://wordpress.org/support/plugin/amp/reviews/#new-post" target="_blank">' . __( 'Review plugin', 'amp' ) . '</a>',
+			'<a href="https://wordpress.org/support/plugin/amp/#new-post" target="_blank">' . __( 'Get support', 'amp' ) . '</a>',
```

Comment:
I think we should swap the order here. Support is more often needed than reviews.

```suggestion
			'<a href="https://wordpress.org/support/plugin/amp/#new-post" target="_blank">' . __( 'Get support', 'amp' ) . '</a>',
			'<a href="https://wordpress.org/support/plugin/amp/reviews/#new-post" target="_blank">' . __( 'Review plugin', 'amp' ) . '</a>',
```